requirements.txt

```diff
@@ -1,36 +1,29 @@
 alembic==1.6.5
 APScheduler==3.8.0
-bcrypt~=3.2.0
 boto3~=1.18.14
-click==7.1.2
+click==8.1.3
```

Comment:
What was the secret to getting this to pass? [I struggled with this yesterday](https://ethyca.slack.com/archives/C029HFVSC8L/p1654638146084379?thread_ts=1654616643.541969&cid=C029HFVSC8L)

Reply:
Looks like `black` was also bumped to v22.3.0 in https://github.com/ethyca/fidesops/pull/614/commits/ada24f8e2ff3ad230c07d4703304ca5bcf4e74fa.